tests/test_mod.py

```diff
@@ -725,6 +735,44 @@ def test_scihub_unresponsive():
             api.download_all(['8df46c9e-a20c-43db-a19a-4240c2ed3b8b'])
 
 
+def test_trigger_lta():
```

Comment:
Maybe split this up into several tests for more granular checks? Like `test_trigger_lta_accepted`, `test_trigger_lta_fails_not_accepted`, ...

Reply:
Also done. Do you think it makes sense to provide a fixture for these very similar unit tests?

Reply:
Yes, good idea. For the `lta_url` and the `api_mock` (or `mock_api`?).

Reply:
I could not think of an elegant way with a fixture. The unit tests for unsuccessful LTA requests are now parameterized with `pytest.mark.parametrize`.

Reply:
nice, thanks!